pkg/operator/controller/certrotation_test.go

```diff
@@ -81,7 +103,10 @@ func getCertConfigAnno(client kubernetes.Interface, namespace, name string) stri
 	s, err := client.CoreV1().Secrets(namespace).Get(context.TODO(), name, metav1.GetOptions{})
 	Expect(err).ToNot(HaveOccurred())
 	val, ok := s.Annotations["operator.cdi.kubevirt.io/certConfig"]
-	Expect(ok).To(BeTrue())
+	Expect(ok).To(BeTrue(), func() string {
+		secretJSON, _ := json.MarshalIndent(s, "", "    ")
+		return string(secretJSON)
```

Comment:
pardon my ignorance but what is the side effect of this function here? What happens to the result?

Reply:
that's fine it's not common in our projects, it's evaluated in case of failure to allow for a more descriptive error msg https://github.com/kubevirt/containerized-data-importer/blob/673e15a77170d1be9ba9bc554b032f4368b67e1a/vendor/github.com/onsi/gomega/gomega_dsl.go#L497-L508